internal/controller/scheduler/scheduler.go

```diff
@@ -28,6 +28,12 @@ var (
 	workersStat = promauto.NewGaugeVec(prometheus.GaugeOpts{
 		Name: "orchard_workers",
 	}, []string{"status"})
+	workersStatus = promauto.NewGaugeVec(prometheus.GaugeOpts{
```

Comment:
What do you think about simply adding a new dimension to the `orchard_workers` metric instead of creating a new one?

Currently `orchard_workers` only has a `status` label, but we can add the `worker_name` too.

Reply:
I was going to go down that path initially, but then I thought that might break setups for existing metric users, as it will go from a counter to a gauge exporting a totally different metric

Reply:
>will go from a counter to a gauge exporting a totally different metric

I think the only counter we have right now is the `orchard_scheduler_loop_iteration_total`. The `orchard_workers` is already a gauge.

Or I'm missing something?

Reply:
Oh interesting, yeah I misread that line. But yeah we'd still be changing what that metric exports - but if you don't think that's an issue I'm happy doing your suggestion. 

It's going to go from an increasing number, to either a 0 or 1 which would still be a breaking change?

Reply:
I think this should only be an UI/UX issue if the consumers have plotted unaggregated graphs (e.g. `orchard_workers{}` in Grafana).

One more thing I've realized is that we have both the Prometheus and OpenTelemetry in Orchard 😀

@fkorotkov should we deprecate the https://github.com/cirruslabs/orchard/pull/82 and only make changes to the OpenTelemetry instrumentation?

Reply:
Cool, I've changed my PR to your suggestion

Reply:
@edigaryev yeah, OTEL all the way.

Reply:
@mcmarkj are you planning to use Prometheus specifically, or OpenTelemetry works for you too?

Reply:
We'd be using Prometheus